src/modules/commander/Arming/HealthFlags/HealthFlags.h

```diff
@@ -77,6 +77,7 @@ struct subsystem_info_s {
 	static constexpr uint64_t SUBSYSTEM_TYPE_PREARM_CHECK = 1 << 28;
 	static constexpr uint64_t SUBSYSTEM_TYPE_OBSTACLE_AVOIDANCE = 1 << 29;
 	static constexpr uint64_t SUBSYSTEM_TYPE_PARACHUTE = 1ULL << 32;
+	static constexpr uint64_t SUBSYSTEM_TYPE_OPEN_DRONE_ID = 1ULL << 33;
```

Comment:
MAVLink needs a corresponding PR to add bitmask field for MAV_SYS_STATUS_SENSOR_EXTENDED